templates/staff/dashboards/repos_with_multiple_projects.html

```diff
@@ -0,0 +1,123 @@
+      <li class="breadcrumb-item active" aria-current="page">
+        Repos with multiple projects
+      </li>
+    </ol>
+  </div>
+</nav>
+{% endblock breadcrumbs %}
+
+{% block jumbotron %}
+<div class="jumbotron jumbotron-fluid jumbotron--danger pt-md-2">
+  <div class="container">
+    <h1 class="display-4">Repos with multiple projects</h1>
+    <p class="lead">
+      Repos with multiple projects on OpenSAFELY
+    </p>
+
+    <p>Remaining: {{ repos|length }}</p>
+  </div>
+</div>
+{% endblock jumbotron %}
+
+{% block staff_content %}
+<div class="container-fluid">
+  <div class="row">
+    <div class="col">
+      <button type="button" class="reset btn btn-outline-primary mb-3" data-column="0" data-filter="">Reset filters</button>
+      <div class="table-responsive">
+        <table class="table table-striped table-sm table--repo">
+          <thead>
+            <tr>
+              <th>Repo</th>
+              <th>Projects</th>
+              <th>Workspaces</th>
+              <th class="text-nowrap">Files released to GitHub</th>
+            </tr>
+          </thead>
+          <tbody>
+            {% for repo in repos %}
+            <tr>
+              <td><a href="{% url 'staff:repo-detail' repo_url=repo.quoted_url %}">{{ repo.name }}</a></td>
+
+              <td>
+                <details>
+                  <summary>
+                    <span class="summary--show">Show</span>
+                    <span class="summary--hide">Hide</span>
+                    {{ repo.projects|length }} projects
+                  </summary>
+                  <ul class="mt-1 mb-0 pl-2 ml-2">
+                    {% for project in repo.projects %}
+                    <li><a href="{{ project.get_staff_url }}">{{ project.name }}</a></li>
+                    {% endfor %}
+                  </ul>
+                </details>
+              </td>
+
+              <td>
+                {% if repo.workspaces|length > 1 %}
+                  <details>
+                    <summary>
+                      <span class="summary--show">Show</span>
+                      <span class="summary--hide">Hide</span>
+                      {{ repo.workspaces|length }} workspaces
+                    </summary>
+                    <ul class="mt-1 mb-0 pl-2 ml-2">
+                      {% for workspace in repo.workspaces %}
+                      <li>
+                        <a href="{{ workspace.get_staff_url }}">
+                          {{ workspace.name }}
+                        </a>
+                      </li>
+                      {% endfor %}
+                    </ul>
+                  </details>
+                {% else %}
+                  {% for workspace in repo.workspaces %}
+                    <a href="{{ workspace.get_staff_url }}">
+                      {{ workspace.name }}
+                    </a>
+                  {% endfor %}
+                {% endif %}
+              </td>
+
+              {% if repo.has_github_outputs %}
```

Comment:
Do we need to know whether a repo has github outputs here? Does it change anything about the data tidying process?

Reply:
I don't _think_ so, but we store it in the database so I figured hey why not!